### PR TITLE
add nowait option to reset

### LINKED ISSUE #45: Allow the reset command to be executed in the background
### Expected Behaviour
Triggering the `aio aem:rde reset` command in a CI / CD pipeline is commonly the last command to be executed. The build status could already be set to succeeded or failed independently of the reset of the environment. It should be possible to trigger a reset without waiting for the finishing of that reset

### Actual Behaviour
Currently the reset command is always blocking

### PR DESCRIPTION
**Pull request by RemoLiechti**
## Description

introduces a --nowait option for 'aio aem rde reset'

## Related Issue

Fixes #45 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by angulito — APPROVED** [review]

File: src/commands/aem/rde/reset.js [hunk 1]
```diff
@@ -37,6 +46,16 @@ class ResetCommand extends BaseCommand {
 Object.assign(ResetCommand, {
   description: 'Reset the RDE',
   args: [],
+  flags: {
+    nowait: Flags.boolean({
```
Comment (jsedding) [inline src/commands/aem/rde/reset.js:50]: Could we please call this flag `wait` with a default of `true`, and add `allowNo: true` to the flag's configuration. This results in a flag called `--no-wait`, which makes for a consistent style. Note, the "log" command has a `--no-color` flag configured in the same way.

File: src/commands/aem/rde/reset.js [hunk 2]
```diff
@@ -37,6 +46,16 @@ class ResetCommand extends BaseCommand {
...
+      description:
+        'Do not wait for the environment to be reset. Check using status command for progress.',
```
Comment (jsedding) [inline src/commands/aem/rde/reset.js:52]:
```suggestion
        'Do not wait for completion of the reset operation. Progress can be manually checked using the "status" command.',
```

File: src/commands/aem/rde/reset.js (hunk 1 above)
Comment (RemoLiechti) [reply src/commands/aem/rde/reset.js:50]: agree. as this was merged already, I will open another PR and add you there.

Comment (RemoLiechti) [reply src/commands/aem/rde/reset.js:50]: see https://github.com/adobe/aio-cli-plugin-aem-rde/pull/73

**Pull request merged**